lit_gpt/lora.py

```diff
@@ -87,6 +88,71 @@ def __init__(self, r: int, lora_alpha: int, lora_dropout: float, merge_weights:
         self.merge_weights = merge_weights
 
 
+class Linear(nn.Linear, LoRALayer):
```

Comment:
I suggest renaming this `LoRALinear` and what's currently `MergedLiner` to `LoRAQKVLinear`.

Then `LoRAQKVLinear` could subclass `LoRALinear` to reuse some of the code, for example `reset_parameters` is the same.

Reply:
Done.